utils/ahp.py

```diff
@@ -1,29 +1,18 @@
+import sys
+
 import numpy as np
 import random
+from typing import Dict, List
+import sys
 
 
-class AnalyticalHierarchyProcess:
-    def __init__(self):
-        self.criteria = ["time", "cost", "lifespan"]
-        self.pets = ["cat", "dog", "snake"]
-        self.criteria_matrix = np.ones([len(self.criteria), len(self.criteria)])
-        self.pets_matrices = [np.ones([len(self.pets), len(self.pets)]) for _ in self.criteria]
-
-    def add_pets_matrix(self, criteria: str, matrix: np.ndarray):
-        self.pets_matrices[self.criteria.index(criteria)] = matrix
-
-    def add_criteria_matrix(self, matrix: np.ndarray):
-        self.criteria_matrix = matrix
-
-    def choose_pet(self) -> str:
-        criteria_vector = calculate_priority_vector(self.criteria_matrix).astype('float64')
-        rank_vector = np.zeros(len(self.pets))
-        for i in range(len(self.pets_matrices)):
-            rank_vector += calculate_priority_vector(self.pets_matrices[i]).astype('float64') * criteria_vector[i]
-
-        print(rank_vector)
-
-        return self.pets[rank_vector.argmax()]
+def process_matrix(raw_matrix: List[List]):
```

Comment:
```suggestion
def process_matrix(raw_matrix: list[list[float]):
```
Since [Python 3.9](https://docs.python.org/3/whatsnew/3.9.html) there's no need to import types for basic data structures like `list` or `dict`.